src/style.css

```diff
@@ -0,0 +1,65 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.search__form {
+  margin-top: 20px;
+  position: relative;
+}
+
+.search__input {
+  width: 100%;
+  height: 70px;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px
+    rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  padding-left: 62px;
+  font-family: "Avenir", sans-serif;
```

Comment:
add font-family to html tag, here you can use inherit

Reply:
after I've moved font-family to html tag, 4 of the tests failed.
![image](https://user-images.githubusercontent.com/105542865/184643920-fd8da845-42d2-41f6-b452-8a5fc01b4a6c.png)
![image](https://user-images.githubusercontent.com/105542865/184644031-8349543d-d439-4f87-8a29-4ce28bf263c6.png)

Reply:
here is the answer)
https://github.com/mate-academy/layout_search-bar-airbnb/pull/1782#discussion_r945772294